rpc_jobs/single_use_slave_template.yml

```diff
@@ -1,58 +0,0 @@
-- job:
```

Comment:
I think the single use slave template still has some value as an example. So its enough to remove the project builds job.